### PR TITLE
Use fluid layout

### PR DESCRIPTION
**Pull request by Nerian**
This pull changes the basic layout to a fluid one. This fixes a problem in 0.0.21 where the contents appear misplaced.

### CODE REVIEW

**Comment by Pajk** [comment]
@Nerian Merged after more than 2 years :tada:. I don't see big difference but looks like the fluid layout looks better on mobile phones. We will see if it breaks something but I hope not. Most of the users use custom layout anyway. Your pull request just deserved to be merged;).

**Comment by Pajk** [comment]
Merged in https://github.com/Apipie/apipie-rails/pull/409

File: app/views/layouts/apipie/apipie.html.erb [hunk 1]
```diff
@@ -1,26 +1,24 @@
...
+    <%= Apipie.include_stylesheets %>
+    <link type='text/css' rel='stylesheet' href='<%= Apipie.full_url("stylesheets/application.css")%>'/>
+    <!-- IE6-8 support of HTML5 elements -->
+    <!--[if lt IE 9]>
+      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
+    <![endif]-->
+  </head>
+  <body>
+    <div class="container-fluid">
+      <div class="row-fluid">
```
Comment (Nerian) [inline app/views/layouts/apipie/apipie.html.erb]: Well this bit is deprecated now. It should be `row` instead of row-fluid. :)

Comment (Nerian) [inline app/views/layouts/apipie/apipie.html.erb]: Oh never mind, the bootstrap included is version 2, for which this syntax is correct.

Comment (Pajk) [inline app/views/layouts/apipie/apipie.html.erb]: Yup I'm not going to update bootstrap version at this point.

**Pull request merged**